data/mods/MindOverMatter/items/armor/head.json

```diff
@@ -15,6 +14,7 @@
     "color": "blue",
     "material_thickness": 1,
     "flags": [ "DIMENSIONAL_ANCHOR" ],
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "mutations": [ "ANCHORCROWN_NOSPELL" ] } ] },
```

Comment:
Are you sure it works? I recall a longstanding issue, that mutations, given by enchantment (which relic_data is) is shown in @ menu, but have no effect whatsoever

Reply:
I did test it and it worked. Wearing the crown prevented me from using any spell at all, then removing it gave me my spellcasting back

Reply:
Working on #68140, it looks like enchantment mutations given by items work, but enchantment mutations given by effects do not.